### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Your Subscriptions_Screen Reader: Narrator is not reading the checked/unchecked status for checkboxes after pressing space

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4288]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS1.3.1; #ScreenReader; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev1; #Narrator 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**Screen Reader:** Narrator 

**MAS Violated:** MAS1.3.1

**NOTE:** Same issue is observed in the below screen
"Tools > Azure > Sign-in > Service Principle > New > Copy&Open > Create Authentication Files

**Repro Steps:**
1.Start Screen Reader. Open Java EE - Eclipse application
2.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
3.Tab till "Tools > Azure > Sign-n" in the navigation bar and press enter
4.Tab till "Device Login" radio button and select it 
5.Tab till "Sign-in" button and press enter
6.Tab till "Copy&Open" button and press enter 
7.Tab till "Enter Code" text box and paste the copied code. Tab till "Next" button and press enter
8.Tab till the required account and press enter
9.Go back to Eclipse IDE
10.Tab till any check-box present against Subscription Name and press space to check/uncheck it
11.Verify if the screen reader is announcing the checked/unchecked status after performing the operation

**Actual Result:**
Narrator is not reading the checked/unchecked status for checkboxes after pressing space

**NOTE:**  NVDA is reading the checked/unchecked status for checkboxes after pressing space

**Expected Result:**
Narrator should read the checked/unchecked status for checkboxes after pressing space

**User impact:**
Screen Reader users will not be notified if the operation is performed or not

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/doc2.aspx?sourcedoc=%7B97358E88-4367-4121-BF9D-F0C6597D03B7%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships%20.docx&action=default&mobileredirect=true&cid=8b75f338-0916-41c9-9fdf-ddb91b2b7698)

[Bug_4288_Java Tooling_Your Subscriptions_Check box_Narrator.pptx](https://github.com/microsoft/azure-tools-for-java/files/4462358/Bug_4288_Java.Tooling_Your.Subscriptions_Check.box_Narrator.pptx)

![Bug_4288_Java Tooling_Your Subscriptions_Check box_NVDA_No Issue](https://user-images.githubusercontent.com/41108985/79005707-19a73500-7b75-11ea-822b-5282b54ddbc3.png)

**Comment by Flanker32** [comment]
@soumyatippapuram This seems to be an issue of Eclipse or narrator, narrator behaves the same in all tables with checkbox in Eclipse, for instance, Windows -> Preferences->Server->Audio, the check status could be read by NVDA but narrator will not.

**Comment by Flanker32** [comment]
* Importance of experience: Low, users may not get response after press space with narrator
* Quality of experience: Low, users could change the screen reader to NVDA which could read the full status
* Alternate solutions:  Use NVDA instead of narrator
* Evidence: All evidence is provided in this issue.
* [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@soumyatippapuram This issue has been linked to AAT, please have a review, thanks!

**Comment by Alexandria-R** [comment]
Approved to close per one free screen reader rule

**Comment by soumyatippapuram** [comment]
Closing this bug as per above comments

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749657
